src/pint/toa.py

```diff
@@ -103,6 +103,7 @@ def get_TOAs(
     bipm_version=None,
     include_gps=None,
     planets=None,
+    include_pn=True,
```

Comment:
Is there any need for this? The column can easily be removed with `remove_pulse_numbers`.

Reply:
especially for reading, I think it's needed.  this directly addresses the use case Deborah had where multiple files are being read in together so you can't remove pulse numbers individually.

Reply:
I think I'm with @aarchiba on this one.  That seems like a really specific and unneeded option.  Because even if you read the files together, you still need to merge them, and so can remove pulse numbers before you do that.  Unless I'm missing something...

Reply:
Merging is handled within `get_TOAs()`.  I guess with this now inserting pulse numbers by default then it's fine to remove this option, although it may be that instead the `strict` option should be used instead.

Reply:
Maybe a `strict` option isn't needed, but I personally prefer to be able to control if the data input will be additionally modified on read.